python/lsst/sims/photUtils/Photometry.py

```diff
@@ -15,458 +15,110 @@
 from collections import OrderedDict
 import lsst.utils
 from lsst.sims.photUtils import Sed, Bandpass, LSSTdefaults, calcGamma, \
-                                calcMagError_m5, PhotometricParameters, magErrorFromSNR
+                                calcMagError_m5, PhotometricParameters, \
+                                magErrorFromSNR, BandpassDict
 from lsst.sims.utils import defaultSpecMap
 
-__all__ = ["PhotometryHardware", "PhotometryBase"]
+__all__ = ["loadBandpassesFromFiles", "loadTotalBandpassesFromFiles"]
```

Comment:
I personally like them as free-floating utility functions.  It means that, whenever the BandpassDict constructor is called, you actually get a BandpassDict (rather than an empty BandpassDict waiting to be initialized with loadBandpassesFromFiles).

I don't want them to belong to any class, because I would like users to be able to call them without having to instantiate any class object.  (The way, in the past, you had to calculate magnitudes using PhotometryBase().manyMagCalc_list(sed))